Astronomy/Astronomy/Operations/SSSFetchImageOperation.h

```diff
@@ -0,0 +1,22 @@
+//
+//  SSSFetchImageOperation.h
+//  Astronomy
+//
+//  Created by Shawn Gee on 5/19/20.
+//  Copyright © 2020 Swift Student. All rights reserved.
+//
+
+#import "SSSConcurrentOperation.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SSSFetchImageOperation : SSSConcurrentOperation
+
+@property NSURL *imageURL;
+@property NSData *imageData;
```

Comment:
Same thing here; might want to make `imageData` readonly, and possibly add an `error` property in case the fetch passed back a networking error.